docs/user-guide/install.md

```diff
@@ -1,9 +1,46 @@
 # Debian 12 Install
 
-These instructions are for installing ASL3 on various x86 systems. AllStarLink v3 is supported on Debian 12 Bookworm systems, and those based on Bookworm (e.g. Raspberry Pi OS). Both the x86_64/amd64 and arm64/aarch64 platforms are supported through apt/deb installation packages.
-
-Note that currently the project does not support armv7l/armhf platforms because all known use of AllStarLink is on hardware which supports the Bookworm arm64 distribution such as Raspberry Pi 3, 4, 5, and Zero 2 W. If you have a platform that must use armv7l/armhf 32-bit packages only please file an issue at [ASL3 on GitHub](https://github.com/AllStarLink/ASL3/issues).
-
+These instructions are for installing ASL3 on general purpose operating systems manually. For
+installation in Raspberry Pis, consider using the [AllStarLink 3 Pi Appliance](pi-detailed.md)
+
+!!! note "Architecture Support"
+    Currently the project does not support armv7l/armhf platforms because all known use of AllStarLink is on hardware which supports the Bookworm arm64 distribution such as Raspberry Pi 3, 4, 5, and Zero 2 W. If you have a platform that must use armv7l/armhf 32-bit packages only please file an issue at [ASL3 on GitHub](https://github.com/AllStarLink/ASL3/issues).
```

Comment:
1. Currently, (add the comma)
2. all known uses (add an "s")
3. "must use armv7l/armhf 32-bit packages only" --> "can only use armv7l/armhf 32-bit packages" ?

Reply:
Fixed